daemons/fenced/fenced_commands.c

```diff
@@ -974,7 +975,7 @@ map_action(GHashTable *params, const char *action, const char *value)
  * \brief Create device parameter table from XML
  *
  * \param[in]     name    Device name (used for logging only)
- * \param[in,out] params  Device parameters
+ * \param[in,out] dev     XML containing device parameters
```

Comment:
Can this just be `[in]`? `xml2list()` is const and the result is a hash table of copied name-value strings.

Reply:
Yep, not sure how I missed that

Reply:
Ah, it was already marked in,out, and I assumed that was correct

Reply:
Looks good to me otherwise

Reply:
Rebased on current main and updated this, no other changes